cmd/api/src/database/sso_providers.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2024 Specter Ops, Inc.
+//
+// Licensed under the Apache License, Version 2.0
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package database
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/specterops/bloodhound/src/model"
+)
+
+const (
+	ssoProviderTableName = "sso_providers"
+)
+
+var (
```

Comment:
I did this since we had talked about using the existing SessionAuthProviders, but since we have the enum in our database, I wanted to have a few constants that can map to that enum as well

Reply:
If we make the enum an int and match the below could we just continue to use the existing constant? 🤔

Reply:
I don't think there's a way to actually make an `int` enum in Postgres, unfortunately.
Open to ideas, though!